### PR TITLE
API Fixes for q4 sprint

### PR DESCRIPTION
**Pull request by hstove**
This PR contains back-end changes related to this milestone:

https://app.zenhub.com/workspaces/blockstack-engineering-582a2234b399ebbf18315399/board?milestones=Q4-%20Explorer%20Sprint%201%23&filterLogic=any&repos=180429576,56251702&showEstimates=false&showPipelineDescriptions=false

- Calculate total locked, unlocked, and vesting total from DB (instead of error-prone aggregations)
- Return the vesting schedule in date form for 2019 investors (which triggers the display of the vesting chart)
- Calculate token grants, if an account was granted tokens with no vesting

---

## Notes for reviewers

First, review the 'local development' part of the README for this project. First, create a `.env` file, and add the Postgresql config from the README. Then, get the relevant Bitcore configuration from me, privately.

Next, install dependencies with `yarn`.

Then, you'll need to seed your database. This might take a while, like roughly 30 minutes. Simply run `yarn seed`, and it'll get a snapshot from Blockstack core and seed your Postgres DB.

Once that's done, you can run the API server with `yarn dev`. If you want to play around and not have to restart the server manually for every code change, you can use `yarn nodemon`.

When it's running, you can access the API at port 4000. The only API effected in this PR is the Stacks address API: `http://localhost:4000/api/stacks/:address`. You can review some of the tickets in [our milestone](https://app.zenhub.com/workspaces/blockstack-engineering-582a2234b399ebbf18315399/board?milestones=Q4-%20Explorer%20Sprint%201%23&filterLogic=any&repos=56251702,180429576&showEstimates=false&showPipelineDescriptions=false) too see some of the ones that we've had problems with.

Although the focus of the code review is on a small part of the app, you should definitely feel free to look around and highlight areas that you think are a problem.

### CODE REVIEW

**Comment by hstove** [comment]
FYI - I made a bunch of `eslint` improvements in #15, but I did it in a separate branch to not ruin to diff of this PR

File: lib/aggregators/stacks-address.ts [hunk 1]
```diff
@@ -44,35 +46,42 @@ class StacksAddress extends Aggregator {
...
     ]);
 
     let unlockInfo = {};
-    if (vestingTotal && vestingTotal > 0) {
+    if (Vesting.vestingTotal && Vesting.vestingTotal > 0) {
       unlockInfo = {
-        formattedUnlockTotal: accounting.formatNumber(vestingTotal * 10e-7),
-        unlockTotalStacks: stacksValue(vestingTotal),
-        unlockTotal: vestingTotal,
+        formattedUnlockTotal: accounting.formatNumber(Vesting.vestingTotal * 10e-7),
```
Comment (zone117x) [inline lib/aggregators/stacks-address.ts]: Dangerous floating point operation here. Should use BigInt w/ polyfill or equivalent

Comment (hstove) [reply lib/aggregators/stacks-address.ts]: Wouldn't this indicate we need to refactor every single place that converts microstacks to Stacks?

Comment (zone117x) [reply lib/aggregators/stacks-address.ts]: Not a blocker for this PR

**Pull request merged**